articles/libraries/machine-learning/load.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Quantum machine learning library
+author: geduardo
+ms.author: v-edsanc@microsoft.com
+ms.date: 02/16/2020
+ms.topic: article
+uid: microsoft.quantum.libraries.machine-learning.load
+---
+
+# Load and classify your own datasets
+
+In this short tutorial we are going to learn how to load your own dataset to train a classifier model with the QDK.
+
+We highly recommend the use of [JSON files](https://en.wikipedia.org/wiki/JSON) to store the data due to its high compatibility with different frameworks like Python and the .NET ecosystem. In particular we recommend to use our template for loading the data, so you can copy-paste the code directly from the samples.  
+
+## Template for loading your datasets
+
+Suppose we have a training dataset $(x,y)$ of size $N=2$ where each instance $x_i$ of $x$ has three features: $x_{i1}$, $x_{i2}$ and $x_{i3}$. The validation dataset has the same structure. Then our `data.json` file would be the following:
+
+```json
+{
+    "TrainingData": {
+        "Features": [
+            [
+                x_11,
+                x_12,
+                x_13
+            ],
+            [
+                x_21,
+                x_22,
+                x_23
+            ]
+        ],
+        "Labels": [
+            y_1,
+            y_2
+        ]
+    },
+    "ValidationData": {
+        "Features": [
+            [
+                xv_11,
+                xv_12,
+                xv_13
+            ],
+            [
+                xv_11,
+                xv_12,
+                xv_13
+            ]
+        ],
+        "Labels": [
+            yv_1,
+            yv_2
+        ]
+    }
+}
+
+```
+
+### Example using the template
+
+Suppose we have a small dataset with the heights and weights of different cats and dogs. This dataset is very small to train a model but will be enough to show the process of loading a dataset.
+
+| Height (m) | Weight (kg) | Animal |
+|-----------|------------|--------|
+| 0.54      | 30         | Dog    |
+| 0.30      | 8          | Cat    |
+| 0.91      | 44         | Dog    |
+| 0.86      | 31          | Dog    |
+| 0.32      | 5         | Cat    |
+| 0.25      | 4          | Cat    |
+

```

Comment:
the columns are offset here...make sure they render correctly in preview

Reply:
I think it works fine